src/main/java/org/resitrack/Main.java

```diff
@@ -72,4 +69,31 @@ public static void managePerson(PersonController personController, Scanner scann
 
         } while (!choice.equals(EXIT));
     }
+
+    public static void manageHouse(Scanner scanner) {
+        String choice;
+        List<House> houses = new ArrayList<>();
+        List<Town> towns = new ArrayList<>();
```

Comment:
why we need create a list of towns object again here ?

Reply:
I fixed it